docs/content/commands/npm.md

```diff
@@ -20,36 +20,42 @@ npm is the package manager for the Node JavaScript platform.  It puts
 modules in place so that node can find them, and manages dependency
 conflicts intelligently.
 
-It is extremely configurable to support a wide variety of use cases.
-Most commonly, it is used to publish, discover, install, and develop node
+It is extremely configurable to support a variety of use cases.  Most
+commonly, you use it to publish, discover, install, and develop node
 programs.
 
 Run `npm help` to get a list of available commands.
 
 ### Important
 
-npm is configured to use npm, Inc.'s public registry at
+npm comes preconfigured to use npm's public registry at
 https://registry.npmjs.org by default. Use of the npm public registry is
-subject to terms of use available at https://www.npmjs.com/policies/terms.
+subject to terms of use available at
+https://www.npmjs.com/policies/terms.
 
-You can configure npm to use any compatible registry you like, and even run
-your own registry. Use of someone else's registry may be governed by their
-terms of use.
+You can configure npm to use any compatible registry you like, and even
+run your own registry. Use of someone else's registry is governed by
+their terms of use.
 
 ### Introduction
 
 You probably got npm because you want to install stuff.
 
-Use `npm install blerg` to install the latest version of "blerg".  Check out
-[`npm install`](/commands/npm-install) for more info.  It can do a lot of stuff.
+The very first thing you will most likely want to run in any node
+program is `npm install` to install its dependencies.
 
-Use the `npm search` command to show everything that's available.
-Use `npm ls` to show everything you've installed.
+You can also run `npm install blerg` to install the latest version of
+"blerg".  Check out [`npm install`](/commands/npm-install) for more
+info.  It can do a lot of stuff.
+
+Use the `npm search` command to show everything that's available in the
+public registry.  Use `npm ls` to show everything you've installed.
 
 ### Dependencies
 
-If a package references to another package with a git URL, npm depends
-on a preinstalled git.
+In addition to the default behavior of installing from the public
+registry, if a package references to another package with a git URL, npm
+will  on a pre-installed git.
```

Comment:
i think a word is missing here, `npm will  on a pre-installed git.`

Reply:
Yeah it wasn't very clear what that was even referring to. I think it's the fact that `git` itself needs to be installed.